test/arrays/jade/defaultValueWithInIntent.chpl

```diff
@@ -0,0 +1,8 @@
+// test from https://github.com/chapel-lang/chapel/issues/11424
```

Comment:
I know there's a lot of examples of it in the test system, but it is generally best to avoid adding test directories with people's names.  It makes it harder to go looking to see if a test already exists along those lines.  We just haven't taken the effort to change the ones that follow this pattern so that they don't any more.

For this one, I'd probably add it to `test/arrays/intents/in/`